### ISSUE TITLE
[Feature Request]: Automatically playing the pronunciation, larger play sound button, and some other UI improvements

### ISSUE THREAD

**Issue by mehdibahrami** [vocably/vocably-pro#23]
Hi Dimah,

Here are some suggestions for improving the user experience:
- The play button is very small; I suggest placing a larger button (even with text) for it in the footer or header.
- I recommend automatically playing the pronunciation in the initial load; this helps the person hear the word and remember its meaning. This feature can be placed in the settings, allowing users to enable or disable it.
- I would suggest changing the homepage setup; it mainly consists of a list of words that users may not necessarily want to see the meanings of immediately, as they might prefer to practice them by clicking the practice button or after a few seconds. My suggestion is to transform the homepage into the practice page and integrate it as a tab in the tab bar. This change would allow for the removal of the back button and the adding a larger play sound button. The second tab could be reserved for looking up and adding new words, the third tab for managing cards (similar to the current edit deck feature), and a fourth tab for settings to manage the current and future settings, such as log out, feedback, etc., it allows you to remove the navigation drawer (hamburger menu).
- I suggest adding a setting where the user can set a specific time in the day to receive notifications from the app to remind them to practice the vocabulary.
- I'm not sure how SRS is currently implemented, but after practicing today's words, when you return to the practice section, you still see the same words! I recommend adding a setting for this or informing the user that they have practiced the words and need to come back tomorrow each time they go to the practice page because I'm not sure how the repetition system is being handled in this case. Additionally, I suggest explaining the functionality of SRS and the intervals implemented in the app on the website to help users better understand it.
- What is the difference between swiping down and to the right? I have never dared to swipe to the right :D But if it serves a different purpose, it might be better to use left and right for correct and incorrect answers and consider swiping up or down for another action.

**Comment by sneas** [comment]
Hey man, thanks a lot for your ideas! I'll eventually split this issue into smaller issues, one item per issue.

You gave it a lot of thought, so I'm having some questions.

> I recommend automatically playing the pronunciation in the initial load

Are you talking about the browser extension?

> I would suggest changing the homepage setup; it mainly consists of a list of words that users may not necessarily want to see the meanings of immediately, as they might prefer to practice them by clicking the practice button or after a few seconds. My suggestion is to transform the homepage into the practice page and integrate it as a tab in the tab bar. This change would allow for the removal of the back button and the adding a larger play sound button. The second tab could be reserved for looking up and adding new words, the third tab for managing cards (similar to the current edit deck feature), and a fourth tab for settings to manage the current and future settings, such as log out, feedback, etc., it allows you to remove the navigation drawer (hamburger menu).

My problem with the current setup is I prefer the app to have a "language context" - it means that I'd prefer users to select what language they want to deal with during the current app session and have this language for:

- practice
- editing cards
- editing the deck settings (coming soon)

I need to know how to handle the language context and the ability for a user to easily switch that context while having multiple tabs related to a single language (1. practice tab and 2. managing cards tab). I expect having the language switch on each tab to be misleading for two reasons:

1. If switching the language in one tab leads to changing the language in the other - this might be unexpected for users
2. Switching the language in every tab independently will be simply annoying to me as a user :)

That's why I'm still inclined to have a "master switch" somewhere in the app. But given your suggestion, I'm not sure where to put it. Have you already though of that?

**Comment by mehdibahrami** [comment]
>Are you talking about the browser extension?

No, I'm talking about the Web/Mobile app, in the practice tab when you load a new vocabulary on the screen you can play the pronunciation.

>That's why I'm still inclined to have a "master switch" somewhere in the app. But given your suggestion, I'm not sure where to put it. Have you already though about that?

I have created some designs (as you can see, my design skills are not the best :D) to illustrate my suggestion. I am considering having a master switch on all Practice, Look up, and Cards pages, which I believe makes sense since users may want to practice the language, add new words to their vocabulary in that language, and manage the words they have added in that specific language. Additionally, having an option to switch languages in the settings seems practical. In the Look up section, I have enabled users to change the language for which they want translations, which I find useful. For instance, if I am learning Spanish, I might want translations in English and Persian, and this approach allows for that flexibility.
I recommend including a switch to change the language (master switch) in the settings as well, along with other settings such as enabling or disabling pronunciation during vocabulary loading, toggling notifications on or off, setting reminder time, options for logging out, deleting accounts, and more.

![Home](https://github.com/vocably/vocably-pro/assets/63710663/d1d15746-67f9-439b-919f-9c559d758df2)

![Home2](https://github.com/vocably/vocably-pro/assets/63710663/b87f9cf3-07d0-4ca2-a30e-44c8b9f49769)

![Look-up](https://github.com/vocably/vocably-pro/assets/63710663/5a85eb86-218c-476c-a06e-31f488734fe6)

![Cards](https://github.com/vocably/vocably-pro/assets/63710663/a9e961ae-6021-461f-9720-f07457837739)

**Comment by sneas** [comment]
Btw, I forgot to mention when you swipe right - you tell the SRS algorithm, "I know this word, and I'm confident". When you swipe down, you tell to the SRS algo "I remembered the word, but it was not easy".
